spec.html

```diff
@@ -31708,7 +31708,11 @@ <h1>Math.atan2 ( _y_, _x_ )</h1>
             1. If _nx_ is *-∞*<sub>𝔽</sub>, return an implementation-approximated Number value representing -π.
             1. If _nx_ is either *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub>, return an implementation-approximated Number value representing -π / 2.
           1. Assert: _nx_ is finite and is neither *+0*<sub>𝔽</sub> nor *-0*<sub>𝔽</sub>.
-          1. Return an implementation-approximated Number value representing the result of the inverse tangent of the quotient ℝ(_ny_) / ℝ(_nx_).
+          1. Let _r_ be the result of the inverse tangent of ℝ(_ny_ / _nx_).
```

Comment:
```suggestion
          1. Let _r_ be the inverse tangent of the quotient ℝ(_ny)_ / ℝ(_nx_).
```

Division should be in the real space.

Reply:
GitHub doesn't let me commit this, so I just committed it locally.

Reply:
Yeah it's unfortunately a long-standing issue that you can't accept suggestions from people who have commit access. It's baffling that it's gone this long without being fixed.

Reply:
`ℝ(_ny)_` should be `ℝ(_ny_)`